src/clusterfuzz/_internal/issue_management/google_issue_tracker/client.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Get a Google Issue Tracker HTTP client."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import os
+import urllib
+import googleapiclient
+import google.oauth2.credentials
+import google_auth_httplib2
+import httplib2
+
+_ROLE_ACCOUNT = "cluster-fuzz-google-issue-tracker"
+_DISCOVERY_URL = "https://issuetracker.googleapis.com/$discovery/rest?version=v1"
+_ISSUE_TRACKER_URL = "https://issuetracker.corp.googleapis.com"
+_UBERPROXY_SERVICE = "up.corp.googleapis.com/"
+_O_AUTH_SCOPE = "https://www.googleapis.com/auth/buganizer"
+_REQUEST_TIMEOUT = 60
+HttpError = googleapiclient.errors.HttpError
+from clusterfuzz._internal.google_cloud_utils import credentials
+
+
+def user():
+    return _ROLE_ACCOUNT + "@google.com"
+
+
+def _request_urllib_for_testing(url, body, method, headers):
+    """Make the request with urllib2 for testing purposes."""
+    # For testing locally with ssolib.
+    from urllib import request
+
+    # Don't accept gzip.
+    if "accept-encoding" in headers:
+        del headers["accept-encoding"]
+    # Pass request to urllib (best effort), which had an opener installed by
+    # ssolib.
+    req = request.Request(url, data=body, headers=headers)
+    req.get_method = lambda: method
+    try:
+        response = request.urlopen(req)
+        content = response.read()
+        info = {
+            "status": response.getcode(),
+            "reason": "OK",
+        }
+        info.update(response.info().items())
+        return httplib2.Response(info), content
+    except Exception as e:
+        return (
+            httplib2.Response(
+                {
+                    "status": e.code,
+                    "reason": e.reason,
+                }
+            ),
+            e.read(),
+        )
+
+
+class _UberProxyHttp(httplib2.Http):
```

Comment:
Is this still required?

Reply:
Agreed that this should be removed (_UBERPROXY_SERVICE as well).

Reply:
Thanks. I also went ahead and removed the commented-out code from build_http, as we can always bring that back in later if we need it, but doesn't seem necessary for now.